Tella/Scenes/Resources/ResourcesView.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  ResourcesView.swift
+//  Tella
+//
+//  Created by gus valbuena on 1/31/24.
+//  Copyright © 2024 HORIZONTAL. All rights reserved.
+//
+
+import SwiftUI
+
+struct ResourcesView: View {
+    @Environment(\.presentationMode) var presentationMode: Binding<PresentationMode>
+    @StateObject private var resourcesViewModel : ResourcesViewModel
+    @EnvironmentObject var sheetManager: SheetManager
+    
+    init(mainAppModel: MainAppModel) {
+        _resourcesViewModel = StateObject(wrappedValue: ResourcesViewModel(mainAppModel: mainAppModel))
+    }
+
+    var body: some View {
+        ContainerView {
+            ZStack {
+                VStack {
+                    resourceHeaderView
+                    
+                    VStack {
+                        DownloadedResources()
+                            .environmentObject(resourcesViewModel)
+
+                        AvailableResources()
+                            .environmentObject(resourcesViewModel)
+                    }.padding(.top, 18)
+                    
+                    Spacer()
+                }.padding(.horizontal, 18)
+            }
+        }
+        .navigationBarHidden(true)
+    }
+    
+    var resourceHeaderView: some View {
```

Comment:
maybe we need to unify the navigation bar because some times we use custom view and sometimes we use native navigation bar

Reply:
Do you mean in the whole app? yeah, it's happening because in the new features we have been adding some constraints to the back button. Since this is not impacting this specific feature I would create a task to refactor the general navigation. What do you think?

Reply:
yeah I meant in the whole app, it's a good idea, 
so we can start creating a centralized nav bar in the next feartures and refactor the old nav bars in the future